components/formats-api/src/loci/formats/FormatTools.java

```diff
@@ -50,13 +50,11 @@
 import loci.formats.meta.MetadataStore;
 import loci.formats.services.OMEXMLService;
 import loci.formats.services.OMEXMLServiceImpl;
-
```

Comment:
Does the whitespace need removing here?  imports from different components are usually separated in bioformats from what I can see.